blocks/footer/footer.js

```diff
@@ -16,5 +16,16 @@ export default async function decorate(block) {
   const footer = document.createElement('div');
   while (fragment.firstElementChild) footer.append(fragment.firstElementChild);
 
+  footer.querySelectorAll('.section.footer-links a').forEach((e) => {
+    e.classList.add(...['link', 'qui-link', 'standalone', 'icon-right']);
```

Comment:
Why not `add('link', 'qui-link', 'standalone', 'icon-right')` instead of spreading an array? Same comment for all occurences.

Reply:
I think the code that's now in main should handle links already? Can you please merge the  branch in yours and see if this code can be removed from footer? I think there might be a problem with the `qui-wc-cta` vs `qui-wc-link` used in main vs. header/footer for which we need to find a solution in both the footer and header (cc @mbradoschi)

Reply:
@andreituicu done 
@mbradoschi i prefer this syntax for readability, not to mention that it can be reused if needed